src/feedback.tsx

```diff
@@ -287,23 +275,8 @@ export const FeedbackScreen: React.FC<FeedbackScreenProps> = ({
                     })}
                 </Box>
             </ResponsiveLayout>
-            {skinName === O2_CLASSIC_SKIN && (
```

Comment:
we may need some of these hacks for the new O2 skin, but I prefer to rethink and reimplement it then